src/components/ScrollButton.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
```

Comment:
[Nitpick] I don't think this is necessary?

```suggestion
```

Reply:
Ur wrong bitsh

Reply:
This button interacts with the user so needs to be a client component. 
`./src/components/ScrollButton.tsx
ReactServerComponentsError:

You're importing a component that needs useState. It only works in a Client Component but none of its parents are marked with "use client", so they're Server Components by default.
Learn more: https://nextjs.org/docs/getting-started/react-essentials

   ,-[/Users/racheljohnson/projects/tidelift-me-up-site/src/components/ScrollButton.tsx:1:1]
 1 | import clsx from "clsx";
 2 | import { useState } from "react";
   :          ^^^^^^^^
 3 | 
 4 | import styles from "./ScrollButton.module.css";
   `----

Maybe one of these should be marked as a client entry with "use client":
  ./src/components/ScrollButton.tsx
  ./src/app/page.tsx
`

Reply:
For anybody finding this later and wondering why Rachel is swearing at me:

1. We were pair-reviewing this PR 
2. I'd commented suggesting removing the `"use client"`, but after trying it, we found it to be a bad idea (I had forgotten it's necessary for client interactivity in RSC, as Rachel pointed out in the previous comment)
3. We were talking around about comment etiquette: how polite to be, and how informative
4. We were joking about being extremely rude in the comments
5. They drafted that "Ur wrong bitsh" comment as a joke
6. I doubled down and said they should 100% post it

I couldn't stop giggling. This was a very positive encounter for everyone involved.